src/stories/KeyboardShortcut.stories.ts

```diff
@@ -0,0 +1,70 @@
+import type { Meta, StoryObj } from '@storybook/vue3'
+import type { ComponentProps } from 'vue-component-type-helpers'
+import { AKeyboardShortcut } from '../components'
+
+type KeyboardShortcutProps = ComponentProps<typeof AKeyboardShortcut>
+
+/**
+ */
```

Comment:
```suggestion
```

Reply:
I meant to add some docs there 🤦‍♀️ , hang on